packages/gdc/src/GDCFeatureDrawerWidget/GDCFeatureDrawerWidget.js

```diff
@@ -0,0 +1,750 @@
+import Divider from '@material-ui/core/Divider'
+import Paper from '@material-ui/core/Paper'
+import { makeStyles } from '@material-ui/core/styles'
+import Table from '@material-ui/core/Table'
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+import Chip from '@material-ui/core/Chip'
+import Tooltip from '@material-ui/core/Tooltip'
+import Link from '@material-ui/core/Link'
+import Icon from '@material-ui/core/Icon'
+import { observer, PropTypes as MobxPropTypes } from 'mobx-react'
+import PropTypes from 'prop-types'
+import React, { useState, useEffect } from 'react'
+import BaseFeatureDetail, {
+  BaseCard,
+} from '@gmod/jbrowse-core/BaseFeatureDrawerWidget/BaseFeatureDetail'
+
+const useStyles = makeStyles(() => ({
+  table: {
+    padding: 0,
+  },
+  title: {
+    fontSize: '1em',
+  },
+  link: {
+    color: 'rgb(0, 0, 238)',
+    'text-decoration': 'underline',
+  },
+}))
+
+/**
+ * Render the consequence table for a simple somatic mutation
+ * @param {*} props
+ */
+function Consequence(props) {
+  const classes = useStyles()
+  const { feature } = props
+  if (!feature.consequence) {
+    return null
+  }
+
+  const consequences = feature.consequence.hits.edges
+
+  return (
+    <BaseCard title="Consequence">
+      <div style={{ width: '100%', maxHeight: 600, overflow: 'auto' }}>
+        <Table className={classes.table}>
+          <TableHead>
+            <TableRow>
+              <TableCell>Gene</TableCell>
+              <TableCell>AA Change</TableCell>
+              <TableCell>Consequence</TableCell>
+              <TableCell>Coding DNA Change</TableCell>
+              <TableCell>Impact</TableCell>
+              <TableCell>Gene Strand</TableCell>
+              <TableCell>Transcript</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {Object.entries(consequences).map(
+              ([key, value]) =>
+                value && (
+                  <TableRow key={key}>
+                    <TableCell component="th" scope="row">
+                      <Link
+                        className={classes.link}
+                        target="_blank"
+                        rel="noopener"
+                        href={`https://portal.gdc.cancer.gov/genes/${value.node.transcript.gene.gene_id}`}
+                      >
+                        {value.node.transcript.gene.symbol}
+                      </Link>
+                    </TableCell>
+                    <TableCell component="th" scope="row">
```

Comment:
There seem to be a lot of cells with `component="th" scope="row"`. Maybe there's something I'm not getting, but I thought that each row should usually only have one cell that's the row table header.

Reply:
Good catch, I did not mean to have those in the non-header rows.